wordpress-mobile/gutenberg-mobile
# Fix for vertical alignment in column that is added from the inserter menu.

Gutenberg PR https://github.com/WordPress/gutenberg/pull/24656

To test:
- Add a Columns block
- Select a verticalAlignment such as middle
- Select a column
- Add a new Column using the block inserter
- The verticalAlignment on the new Column should be set to the same as for Columns.

PR submission checklist:
